modules/phase_field/include/action/BicrystalBoundingBoxICAction.h

```diff
@@ -11,7 +11,7 @@
 #include "Action.h"
 
 /**
- * Automatically generates all variables to model a polycrystal with op_num orderparameters
+ * Bycrystal using a bounding box
```

Comment:
I think I saw that in 4 other spots as well, but one never knows with these scientists ;-)